bot/commands/Configuration/aliasInhibitor.js

```diff
@@ -0,0 +1,18 @@
+const { Inhibitor } = require('klasa');
+
+module.exports = class extends Inhibitor {
+
+    constructor(...args) {
+        super(...args, {
+            name: 'aliasInhibitor'
+        });
+    }
+
+    async run(msg, cmd) {
+        const [alias] = message.content.slice(msg.prefixLength).trim().split(' ');
+        if (msg.guild.settings.get('disabledAliases').includes(alias)) {
+            throw true;
+        }
+        return 'promise?'
```

Comment:
Unsure what this is supposed to be...

Reply:
anything falsy, you could omit this line and it would return undefined which would be sufficient.